### ISSUE TITLE
BSO Elder bow pricing

### ISSUE THREAD

**Issue by Lilbourne** [oldschoolgg/oldschoolbot#3670]
Mylife recently posted a poll, that had very good feedback, for the pricing to be changed on both Elder bow (u) and Elder bow to be more in line with other iterations of bows that already exist. 
![SmartSelect_20220309-211359_Samsung Internet](https://user-images.githubusercontent.com/70830013/158006578-c6d69149-3925-4356-a644-9587b07adc50.jpg)

The current pricing seems to be backwards for the bows.

I am proposing that the price not only be switched, but moved over to alch value as opposed to selling, so there isn't a boatload of gp coming in from easy selloffs. The new pricing would be like
Elder bow (u): High alch: 10k, Sell price: 1025
Elder bow: High alch: 19k, Sell price: 2100

**Comment by themrrobert** [comment]
I can get behind  this.

If my understanding of the code is correct, we should set it like this:
e bow u: alch: 10k, sell: 5k
e bow   :  alch: 19k, sell: 10k
and then the bot will cut the sell price by 75-80% because the sell price is close to/less than the alch price

**Comment by Lilbourne** [comment]
Yeah honestly even that would work. I wasn't exactly sure how the selling price cut worked, but those numbers look good as well.

**Comment by DarkBugz** [comment]
Bso suggestions go to polls
